__tests__/integration.test.js

```diff
@@ -219,6 +219,31 @@ describe("GET requests", () => {
         });
     });
   });
+
+  describe("/api/users", () => {
+    it('should return with status code 200 "OK"', () => {
+      return request(app).get("/api/users").expect(200);
+    });
+
+    it("responds with an array of user objects containing the required properties", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body: { users } }) => {
+          expect(users.length).toBeGreaterThan(0);
```

Comment:
Since we know how many users are in the test data, you can assert the exact number you expect here